ossdbtoolsservice/driver/types/psycopg_driver.py

```diff
@@ -290,7 +291,17 @@ def get_error_message(self, error) -> str:
         """
         Get the message from DatabaseError instance
         """
-        return error.diag.message_primary
+        # If error.diag.message_primary is not None, return it.
+        if error.diag and error.diag.message_primary:
+            return error.diag.message_primary
+
+        # If error.args exists and has at least one element, return the first element as the error message.
+        elif hasattr(error, 'args') and error.args and len(error.args) > 0:
+            return error.args[0]
```

Comment:
Why only first message?

Reply:
The error message is supposed to be in error.diag.message_primary but noticed some of the errors have an error message in args[0]. I'll check whether there can be multiple args and add